src/test/kotlin/net/datafaker/example/SchemaExampleTest.kt

```diff
@@ -0,0 +1,42 @@
+package net.datafaker.example
+
+import net.datafaker.Faker
+import net.datafaker.transformations.Field
+import net.datafaker.transformations.JavaObjectTransformer
+import net.datafaker.transformations.Schema
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.util.*
```

Comment:
Use specific imports?

Reply:
Why?

Reply:
I'm not overly picky about it. Might be a spotless thing, I'd have to double check. I just figured that'd be more consistent, but honestly I didn't check the existing code base to see.

Reply:
Well, it's not me who brought it up :-) I have really no idea, I just use IntelliJ defaults, and that's it.

Reply:
That's fair, if that's how IntelliJ defaults then it likely is like that elsewhere in the code base. We can close this.